src/FirstPayment/FirstPaymentBuilder.php

```diff
@@ -132,6 +131,8 @@ public function create()
         $createMolliePayment = app()->make(CreateMolliePayment::class);
         $this->molliePayment = $createMolliePayment->execute($payload);
 
+        Payment::createFromMolliePayment($this->molliePayment, $this->owner, $this->actions->toMolliePayload());
```

Comment:
See my comment on the `Payment` model. Also, makes sense to rename the `toMolliePayload`, as it's no longer a payload getting sent to Mollie.